### PR TITLE
Gitattribute file + remove unused wsidav config fie

### PR DESCRIPTION
**Pull request by inkhey**
related to #4062.
Just discovering usage of the gitattribute file, i do a proposal for tracim.
Add gitattribute file and remove unused wsgidav file.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works : irrelevant : just dotfile.
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution.
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link) : irrelevant : just dotfile. 
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Code
+*.py text eol=lf diff=python
```
Comment (inkhey) [inline .gitattributes:2]: permit to have better diff for python if version of git contain the diff parser for python.

File: .gitattributes [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+*.html text diff=html
+*.js text
+*.css text diff=css
+*.sh text eol=lf diff=bash
+*.mako text
+*.mak text linguist-language=Mako
```
Comment (inkhey) [inline .gitattributes:8]: force *.mak to be considered as Mako instead of Makefile.

File: .gitattributes [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+*.ini.sample text
+*.in text
+requirements.txt text
+
+# Documentation
+LICENSE text
+CHANGELOG text
+*.md text diff=markdown
+
+# To ignore from release
```
Comment (inkhey) [inline .gitattributes:32]: Just an exemple of usage of export-ignore, not sure we should keep this.

File: .gitattributes [hunk 4]
```diff
@@ -0,0 +1,63 @@
+# Code
+*.py text eol=lf diff=python
+*.html text diff=html
+*.js text
```
Comment (raphj) [inline .gitattributes:4]: No specific support for JS?

File: .gitattributes (hunk 3 above)
Comment (raphj) [reply .gitattributes:32]: I don't think so, especially editor configurations, which is good to share across the team

File: .gitattributes (hunk 4 above)
Comment (inkhey) [reply .gitattributes:4]: as explained there: https://tekin.co.uk/2020/10/better-git-diff-output-for-ruby-python-elixir-and-more
there is currently not diff specific for javascript in latest git : https://github.com/git/git/blob/master/userdiff.c

Comment (inkhey) [reply .gitattributes:4]: note: it's seems possible to had custom one in git.

**Comment by PhilippeAccorsi** [comment]
I think this PR need to be in draft and change wording of title to something like "wip" or "poc" or something explicit (because this is not related to an issue).

**Comment by PhilippeAccorsi** [comment]
I think also its a good idea to create issue and explain why it can be used/interresting for tracim project.

**Comment by inkhey** [comment]
> I think also its a good idea to create issue and explain why it can be used/interresting for tracim project.

i do create a more general issue about packaging strategy and remove the export-ignore from this PR https://github.com/tracim/tracim/issues/4061
I suggest to merge this PR now, as this is just removing a unused file and adding some dot file to make diff a bit better for dev (not strong decision to make here).

**Review by raphj — APPROVED** [review]

**Review by PhilippeAccorsi — APPROVED** [review]

**Pull request merged**